inc/compatibility/base.php

```diff
@@ -194,15 +209,15 @@ protected function run_fix_everytime() {
 	 * @return void
 	 */
 	protected function persist_data( array $plugin_methods = [] ) {
-		$pantheon_applied_fixes                         = get_option( 'pantheon_applied_fixes' ) ?: [];
-		$old                                            = $pantheon_applied_fixes[ static::$plugin_slug ] ?? [];
+		$pantheon_applied_fixes = get_option( 'pantheon_applied_fixes' ) ?: [];
+		$old = $pantheon_applied_fixes[ static::$plugin_slug ] ?? [];
 		$pantheon_applied_fixes[ static::$plugin_slug ] = [
-			'plugin_slug'      => static::$plugin_slug,
-			'plugin_name'      => static::$plugin_name,
-			'plugin_status'    => $plugin_methods ? 'automated' : 'waiting',
-			'plugin_message'   => 'Manual fixes can be safely removed.',
-			'plugin_class'     => static::class,
-			'plugin_methods'   => implode( ',', $plugin_methods ),
+			'plugin_slug' => static::$plugin_slug,
+			'plugin_name' => static::$plugin_name,
+			'plugin_status' => $plugin_methods ? 'automated' : 'waiting',
+			'plugin_message' => 'Manual fixes can be safely removed.',
```

Comment:
Is there a reason why you aren't internationalizing this string?

Reply:
nice catch! thanks